packages/swc-configs/README.md

```diff
@@ -26,50 +26,58 @@ npm install -D @workleap/swc-configs @swc/core @swc/helpers
 
 ### Dev
 
-Create a `swc.dev.ts` file at the root of your project:
+Create a `swc.dev.js` file at the root of your project:
 
 ```
 root
 |---/src
-|---swc.dev.ts
+|---swc.dev.js
+|---webpack.dev.js
 ```
 
-Then, in the newly created file, import and execute the `defineDevConfig` function to return a valid SWC configuration object:
+Then, in the newly created file, import and execute the `defineDevConfig` function to export a valid SWC configuration object:
 
 ```ts
```

Comment:
Since this is a `js` file now, you should colorize for `js`, not `ts` (even if at the end, it should be the same...)